### PR TITLE
Mention to the correct article

### PR DESCRIPTION
**Pull request by Amer-Adel**
The "next" article is not the correct one (variables article).

### CODE REVIEW

**Comment by hamishwillee** [comment]
You've changed "Next" to "Variables", but link to the same article - so the fix is to correctly name the next article?

But the next article in sequence is actually "What_went_wrong" if you click the Next page link on the bottom. This means either the sidebar or this link is wrong. Which do you think it is?

**Comment by Amer-Adel** [comment]
The link of the "next article" is actually point to the variables article, which is wrong, so we just need to correct the name of the link to "variables article", it seems that the article What_went_wrong was added new to the list.

And for the next link at the bottom of the page it is correctly point to the What_went_wrong, Our modification has no effect on it!

File: files/en-us/learn/javascript/first_steps/a_first_splash/index.md [hunk 1]
```diff
@@ -142,7 +142,7 @@ In our example:
 
 - Our final two variables store a guess count of 1 (used to keep track of how many guesses the player has had), and a reference to a reset button that doesn't exist yet (but will later).
 
-> **Note:** You'll learn a lot more about variables and constants later on in the course, starting with the [next article](/en-US/docs/Learn/JavaScript/First_steps/Variables).
+> **Note:** You'll learn a lot more about variables and constants later on in the course, starting with the [variables article](/en-US/docs/Learn/JavaScript/First_steps/Variables).
```
Comment (hamishwillee) [inline files/en-us/learn/javascript/first_steps/a_first_splash/index.md:145]: The style guidelines suggest referencing  by the linked topic name. 
```suggestion
> **Note:** You'll learn a lot more about variables and constants later on in the course, starting with the article [Storing the information you need — Variables](/en-US/docs/Learn/JavaScript/First_steps/Variables).
```

**Review by hamishwillee — APPROVED** [review]
> Our modification has no effect on it!

Doh! You're absolutely right. Thank you.

**Pull request merged**